__mocks__/openmrs-esm-module-config.mock.tsx

```diff
@@ -1,32 +1,13 @@
-import React from "react";
+import React from 'react';
+import { ConfigMock } from './openmrs-esm-config.mock';
```

Comment:
This is very confusing, since there is no module called `@openmrs/esm-config`. I would just put that mock in this file. No reason to make people click around to find it. You can add it as an export to this file if you like.